### PR TITLE
Separate approved for supply

### PR DESCRIPTION
**Pull request by fgregoribattista**
separate approval action in supply

### CODE REVIEW

File: src/components/common/Approve.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="supply-success">
```
Comment (pmprete) [inline src/components/common/Approve.vue:2]:
```suggestion
  <div class="approve">
```
Try using the same name as the file but with dashes

File: src/middleware/market.js [hunk 2]
```diff
@@ -650,7 +633,13 @@ export default class Market {
...
   }
 
-  async isAllowance(account, amountBN) {
+  /**
+   * Check if the market is approve
+   * @param account
+   * @param amountBN => amount in BN
+   * @returns bool = false need approve
+   */
+  async isAllowance(account, amountBN = this.getAmountDecimals(1)) {
```
Comment (pmprete) [inline src/middleware/market.js:642]: Shouldn't this be 0?

**Review by pmprete — APPROVED** [review]
Except my comments looks good

File: src/middleware/market.js (hunk 2 above)
Comment (fgregoribattista) [reply src/middleware/market.js:642]: no, because never have to approve amount=0

**Review by pmprete — APPROVED** [review]
Nice Work

**Pull request merged**